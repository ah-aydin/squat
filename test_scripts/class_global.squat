class Person {
    string name;
    int age;
    int height;
}

Person global_person = Person("global guy", 99, 200);

func main() {
    println("The name of the global person is: " + global_person.name);
}
