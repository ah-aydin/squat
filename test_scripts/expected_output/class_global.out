Compiling and running file: test_scripts/class_global.squat
The name of the global person is: global guy
